network-api/networkapi/templates/pages/buyersguide/editorial_content_index_page.html

```diff
@@ -2,78 +2,85 @@
 {% load i18n localization static wagtailcore_tags wagtailimages_tags %}
 
 {% block guts %}
-  <div class="tw-container tw-my-6">
-    <h1>{{ page.title }}</h1>
+  <div class="tw-relative">
+    <div class="tw-container tw-my-6 tw-relative tw-z-10">
```

Comment:
Can you add some context why we need these two elements to be relative? It seems like the outer is only wrapping the inner.